go/gosec/bad_imports/unsafe.yaml

```diff
@@ -0,0 +1,14 @@
+rules:
+  - id: avoid-using-unsafe-pkg
+    message:
+      Package `unsafe` contains operations that step around the type safety of Go programs.
+      Packages that import unsafe may be non-portable and are not protected by the Go 1 compatibility guidelines.
+    metadata:
+      cwe: 'CWE-242: Use of Inherently Dangerous Function'
+      source_rule_url: https://github.com/securego/gosec
```

Comment:
Can we standardize on whether we use hyphens '-' or underscores '_'?

I think we use hyphens everywhere else. Is there a parsing reason we use underscores here?

Reply:
Thanks for the flag. I wish we can enforce it during the rule writing process. I will address all the renaming of `source_rule_url ` in a separate PR. Sounds good?

Reply:
Sounds good!